packages/flutter/lib/src/material_animated_icons/animated_icons.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of material_animated_icons;
+
+// The code for drawing animated icons is kept in a private API, as we are not
+// yet ready for exposing a public API for (partial) vector graphics support.
+// See: https://github.com/flutter/flutter/issues/1831 for details regarding
+// generic vector graphics support in Flutter.
+
+/// Shows an animated icon at a given animation [progress].
+///
+/// The available icons are specified in [AnimatedIcons].
+class AnimatedIcon extends StatelessWidget {
+
+  const AnimatedIcon({
+    @required this.progress,
+    @required this.color,
+    @required this.icon,
+    this.semanticLabel,
+    this.textDirection,
+    // TODO(amirh): add a parameter for controlling scaling behavior.
+  }) : assert (progress != null),
```

Comment:
nit: extraneous space after `assert`. (try running `dartfmt`? it might work well enough for this file)